code/modules/mob/living/carbon/carbon.dm

```diff
@@ -260,19 +260,28 @@
 /mob/living/carbon/proc/help_shake_act(mob/living/carbon/M)
 	if(src == M)
 		return
-	var/t_him = "it"
+	var/t_him = "them"
 	if(gender == MALE)
 		t_him = "him"
 	else if(gender == FEMALE)
 		t_him = "her"
```

Comment:
we could probably just use p_them() for this but it's oos if you don't wanna

Reply:
doing